ml-agents-envs/mlagents_envs/logging_util.py

```diff
@@ -12,7 +12,8 @@
 _loggers = set()
 _log_level = NOTSET
 DATE_FORMAT = "%Y-%m-%d %H:%M:%S"
-LOG_FORMAT = "%(asctime)s %(levelname)s [%(filename)s:%(lineno)d] %(message)s"
+DEBUG_LOG_FORMAT = "%(asctime)s %(levelname)s [%(filename)s:%(lineno)d] %(message)s"
```

Comment:
BTW, I think this was added by Anupam for log parsing (so he could get the time). Just checking in with @hvpeteet to make sure we aren't using it anywhere in cloud.

Reply:
We don't parse logs in any meaningful way at the moment so I think this should be fine. We essentially just record stdout and stderr and forward it on to the user. This may also get forwarded to stackdriver but each line would already have a timestamp provided upon upload.

Reply:
Is there a way to provide --debug through the YAML? Just worried if users are depending on the current logs and want to restore their old functionality.

Reply:
YES ! 
You can do this by adding `debug: true` in the yaml. For instance : 

```
debug: true
behaviors:
  3DBall:
    trainer_type: ppo
    hyperparameters:
      batch_size: 64
      buffer_size: 12000
      learning_rate: 0.0003
      beta: 0.001
      epsilon: 0.2
      lambd: 0.99
      num_epoch: 3
      learning_rate_schedule: linear
    network_settings:
      normalize: true
      hidden_units: 128
      num_layers: 2
      vis_encode_type: simple
    reward_signals:
      extrinsic:
        gamma: 0.99
        strength: 1.0
    keep_checkpoints: 5
    max_steps: 500000
    time_horizon: 1000
    summary_freq: 12000
    threaded: true

```

The use case is not documented, I will add a line in the changelog.

Reply:
Thanks